src/main/java/Duke.java

```diff
@@ -1,10 +1,108 @@
+import java.util.Scanner;
+
 public class Duke {
```

Comment:
Consider encapsulating the handling of Tasks under a new class called TaskManager to handle add, get, sets, mark done. This allows for better abstraction of data and functions. This will allow Duke to handle more of the CLI matters such as printing and handling user inputs.

Reply:
Thanks for the great idea. Will consider doing it